src/xdocs/google_style.xml

```diff
@@ -67,19 +56,29 @@
       <subsection name="Coverage table" id="Google_Coverage_table">
         <p>
           <b>ATTENTION:</b>
```

Comment:
I question if these paragraphs should be their own sections similar to Legend. They are more telling about how to read the table, then actually supporting the information within it.

Reply:
we can create a new section named `ATTENTION`. Or can you suggest some better name?

@romani can you also share your opinion

Reply:
section that is named "Attention" is weird to my mind.
@rnveach, do you have good name proposal ?

Reply:
It seems `Useful information` would probably fit this.

Reply:
@Zopsss , please use this name for new section

Reply:
Not new.